extension/contextmenu/colormode.js

```diff
@@ -0,0 +1,86 @@
+const storagekey = 'visbug-color-mode'
+const defaultcolormode = 'hsla'
+
+const color_options = [
+  'hsla',
+  'hex',
+  'rgba',
+  // 'hsv',
+  // 'lch',
+  // 'lab',
+  // 'hcl',
+  // 'cmyk',
+  // 'gl',
+  // 'as authored',
+]
+
+const colormodestate = {
+  mode: defaultcolormode
+}
+
+const sendColorMode = () => {
+  chrome.tabs.query({active: true, currentWindow: true}, ([tab]) => {
+    tab && chrome.tabs.sendMessage(tab.id, {
+      action: 'COLOR_MODE',
+      params: {mode:colormodestate.mode},
+    })
+  })
+}
+
+const getColorMode = () => {
+  chrome.storage.sync.get([storagekey], value => {
+    let found_value = value[storagekey]
+
+    const is_default = found_value
+      ? value[storagekey] === defaultcolormode
+      : false
```

Comment:
Rather than fall back to a static, inline value of `false`, I'd recommend importing a well-known global defaults object. I believe this would make it easier to modify defaults in the future and help with reusability in the event that you need to use or compare against the default value in other places.